public/app/features/datasources/components/EditDataSourceActions.tsx

```diff
@@ -35,44 +24,41 @@ export function EditDataSourceActions({ uid }: Props) {
 
   return (
     <>
-      <LinkButton
-        icon="apps"
-        fill="outline"
-        variant="secondary"
-        href={`dashboard/new-with-ds/${dataSource.uid}`}
-        onClick={() => {
-          trackCreateDashboardClicked({
-            grafana_version: config.buildInfo.version,
-            datasource_uid: dataSource.uid,
-            plugin_name: dataSource.typeName,
-            path: location.pathname,
-          });
-        }}
-      >
-        Build a dashboard
-      </LinkButton>
-
       {hasExploreRights && (
-        <LinkButton
-          icon="compass"
-          fill="outline"
+        <Button
+          type="button"
           variant="secondary"
-          className={styles.button}
-          href={constructDataSourceExploreUrl(dataSource)}
+          disabled={!canDelete}
           onClick={() => {
             trackExploreClicked({
               grafana_version: config.buildInfo.version,
               datasource_uid: dataSource.uid,
               plugin_name: dataSource.typeName,
               path: location.pathname,
             });
+            location.href = constructDataSourceExploreUrl(dataSource);
           }}
         >
-          Explore
-        </LinkButton>
+          Explore data
+        </Button>
       )}
-
-      <Button type="button" variant="destructive" disabled={!canDelete} onClick={onDelete} className={styles.button}>
+      <Button
+        type="button"
+        variant="secondary"
+        disabled={!canDelete}
+        onClick={() => {
+          trackCreateDashboardClicked({
+            grafana_version: config.buildInfo.version,
+            datasource_uid: dataSource.uid,
+            plugin_name: dataSource.typeName,
+            path: location.pathname,
+          });
+          location.href = `dashboard/new-with-ds/${dataSource.uid}`;
+        }}
+      >
+        View dashboards
```

Comment:
This does not go to "View dashboards" page but a new dashboard with a a new panel, with this data source instance pre-selected